labelbox/schema/project.py

```diff
@@ -422,6 +422,41 @@ def review_metrics(self, net_score):
         res = self.client.execute(query_str, {id_param: self.uid})
         return res["project"]["reviewMetrics"]["labelAggregate"]["count"]
 
+    def setup_editor(self, ontology):
+        """
+        Sets up the project using the Pictor editor.
+
+        Args:
+            ontology (Ontology): The ontology to attach to the project
+        """
+        labeling_frontend = next(
+            self.client.get_labeling_frontends(
+                where=Entity.LabelingFrontend.name == "Editor"))
+        self.labeling_frontend.connect(labeling_frontend)
+
+        LFO = Entity.LabelingFrontendOptions
+        self.client._create(
+            LFO, {
+                LFO.project:
+                    self,
+                LFO.labeling_frontend:
+                    labeling_frontend,
+                LFO.customization_options:
+                    json.dumps({
+                        "tools": [],
+                        "classifications": []
+                    })
+            })
```

Comment:
There is no harm in adding this but it doesn't seem necessary. @nickroberts404 can I snip this?

Reply:
Hmmm i'd keep it for right now, I want to do an actual thorough cleansing of LFO's from Labelbox and that would be a better time to take this out, there might be some weird gotchas somehwhere or lesser known features that would break unless we really look around